ui/intermittent-failures/helpers.js

```diff
@@ -120,4 +120,24 @@ export const tableRowStyling = function tableRowStyling(state, bug) {
   return {};
 };
 
+export const tableProps = function tableProps() {
```

Comment:
Do we plan on using these functions in other classes?  If not, why not just create them in the ``MainView.jsx`` file?  Or even create them inline at the definition of ``ReactTable``.  I think that would create less noise, tbh.

Reply:
Sure, I have moved them to the `MainView.jsx` file. :)